File: langs/ruby/problems/inventory/stub.rb
class Simulation
  def initialize
  end
  def create_item(sku, name)
    raise 'not implemented'
  end
  def stock(sku, delta)
    raise 'not implemented'
  end
  def get_qty(sku)
    raise 'not implemented'
  end
  def list_low(threshold)
    raise 'not implemented'
  end
  def reserve(sku, n)
    raise 'not implemented'
  end
  def release(sku, n)
    raise 'not implemented'
  end
  def ship(sku, n)
    raise 'not implemented'
  end
end

File: langs/typescript/problems/inventory/stub.ts
class Simulation {
  constructor() {}
  createItem(sku, name) { throw new Error('not implemented'); }
  stock(sku, delta) { throw new Error('not implemented'); }
  getQty(sku) { throw new Error('not implemented'); }
  listLow(threshold) { throw new Error('not implemented'); }
  reserve(sku, n) { throw new Error('not implemented'); }
  release(sku, n) { throw new Error('not implemented'); }
  ship(sku, n) { throw new Error('not implemented'); }
}
module.exports = { Simulation };
